packages/react-app/src/components/Options/styles.module.css

```diff
@@ -35,3 +35,52 @@
   color: var(--backgroundGrey);
   font-weight: 800;
 }
+
+.header_container {
+  display: flex;
+  flex-direction: row;
+}
+
+.advanced_back {
+  background-color: none;
+  border: "none";
+  color: var(--textColorWhite);
+}
+
+.advanced_body {
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.tabs {
+  text-align: center;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.caption_container {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.marker,
+.marker_none,
+.marker_ubiquitous {
```

Comment:
Will need to adjust the z-index on this (or on the tack) to put it below that, while staying above the slider.